Resources/public/js/views/services/plugins/ez-searchplugin.js

```diff
@@ -394,7 +399,9 @@ YUI.add('ez-searchplugin', function (Y) {
                     var content = this._createContent(hit),
                         locationIndex = contentIdsLocationIndexMap[content.get('contentId')];
```

Comment:
What do you think about renaming it to something like `locationIndexes`? It may contain more than one index.